model/host/host.go

```diff
@@ -3729,12 +3729,19 @@ func (h *Host) SetTemporaryExemption(ctx context.Context, exemptUntil time.Time)
 	}
 
 	temporarilyExemptUntilKey := bsonutil.GetDottedKeyName(SleepScheduleKey, SleepScheduleTemporarilyExemptUntilKey)
+	sleepScheduleStartKey := bsonutil.GetDottedKeyName(SleepScheduleKey, SleepScheduleNextStartTimeKey)
```

Comment:
The issue is that the next stop/start won't take effect, nor will they be refreshed as long as the temporary exemption is going, so it might just confuse them if they see the next stop/start is set (or maybe even has long past) while they're still on a temporary exemption.